.github/ISSUE_TEMPLATE/bug_report_template.yaml

```diff
@@ -72,26 +72,33 @@ body:
         mlflow.doctor()
         ```
         Then, make sure the printed out information matches what you expect and paste it (with sensitive information masked) in the box below. If you know the command that was used to launch your tracking server (e.g. `mlflow server -h 0.0.0.0 -p 5000`), please provide it.
+      value: |
+        ```
+        < REPLACE ME (DO NOT REMOVE BACKTICKS) >
```

Comment:
Let me know if there is a better message :)

Reply:
I like it.